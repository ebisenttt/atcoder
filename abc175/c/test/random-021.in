309424482475935 265822996283588 61958957275562
