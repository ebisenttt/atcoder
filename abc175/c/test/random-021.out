61588653373687
